ResearchKit/Common/ORKTaskViewController.m

```diff
@@ -1140,7 +1169,10 @@ - (void)flipToNextPageFrom:(ORKStepViewController *)fromController {
         return;
     }
     
-    ORKStep *step = [self nextStep];
+    ORKStep *step = fromController.parentReviewStep;
+    if (!step) {
+        step = [self nextStep];
+    }
```

Comment:
This happens when a user clears the step result = skips the answer. 
Skipping calls `skipForward` -> `goForward` -> ... -> `flipToNextPageFrom:`

We should leave it as it is and log nothing